src/config.js

```diff
@@ -0,0 +1,19 @@
+const { join } = require('path');
+const { sync } = require('node-read-yaml');
+const { cfgFile, rootDir, fmtProgressBar } = require('./common.js');
```

Comment:
You don't need trailing js extension.

```suggestion
const { cfgFile, rootDir, fmtProgressBar } = require('./common');
```